apps/www/registry/stories/accordion.stories.tsx

```diff
@@ -15,33 +15,40 @@ const meta = {
   title: "ui/Accordion",
   component: Accordion,
   tags: ["autodocs"],
-  argTypes: {},
-  render: (args) => (
-    <Accordion {...args}>
-      <AccordionItem value="item-1">
-        <AccordionTrigger>Is it accessible?</AccordionTrigger>
-        <AccordionContent>
-          Yes. It adheres to the WAI-ARIA design pattern.
-        </AccordionContent>
-      </AccordionItem>
-      <AccordionItem value="item-2">
-        <AccordionTrigger>Is it styled?</AccordionTrigger>
-        <AccordionContent>
-          Yes. It comes with default styles that matches the other components'
-          aesthetic.
-        </AccordionContent>
-      </AccordionItem>
-      <AccordionItem value="item-3">
-        <AccordionTrigger>Is it animated?</AccordionTrigger>
-        <AccordionContent>
-          Yes. It's animated by default, but you can disable it if you prefer.
-        </AccordionContent>
-      </AccordionItem>
-    </Accordion>
-  ),
+  argTypes: {
+    children: {
+      table: {
+        disable: true,
+      },
+    },
```

Comment:
FYI, rather than removing `children` from the table altogether (which can imply that it's an unsupported prop), you can remove just the control like so:

```js
children: {
  control: false,
},
```

Alternatively, where appropriate (e.g. Button), you can constrain the `children` control to just a string:

```js
children: {
  control: 'text',
},
```

Docs: https://storybook.js.org/docs/api/arg-types#control (we could do a better job making this more findable 😅)

Reply:
Which is what I ended up doing cause I regularly use the code snippet to flesh out a component and having to delete some `React.Fragment` tags will feel sloppy 😤

Reply:
I so appreciate how much thought and effort you're putting into making this as useful as possible for your users!